src/main/java/org/sasanlabs/service/impl/EndPointsInformationProvider.java

```diff
@@ -53,15 +52,14 @@ public EndPointsInformationProvider(
     @Override
     public List<AllEndPointsResponseBean> getSupportedEndPoints() throws JsonProcessingException {
         List<AllEndPointsResponseBean> allEndpoints = new ArrayList<>();
-        Map<String, ICustomVulnerableEndPoint> nameVsCustomVulnerableEndPoint =
-                envUtils.getAllClassesExtendingIGetInjectionPayload();
-        for (Map.Entry<String, ICustomVulnerableEndPoint> entry :
-                nameVsCustomVulnerableEndPoint.entrySet()) {
+        Map<String, Object> nameVsCustomVulnerableEndPoint =
```

Comment:
check if scanner endpoint and sitemap endpoints are working properly. I think we need to fix sitemap.xml